### ISSUE TITLE
ubuntu-bug does not work with Pop!_OS

### ISSUE THREAD

**Issue by sramkrishna** [pop-os/pop#113]
If this is a bug, please use the template below. If this is a question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**
A Pop!_OS user reported when trying out Pop!_OS that he had some trouble with Pop!_OS starting up.  He attempted to use the ubuntu-bug command to file a bug but it said that it was not a Pop!_OS package.  Do we support submitting bugs through ubuntu-bug.  Can the script be modified to file bugs?

I assume part of the problem is that ubuntu-bug probably uses launchpad to create bugs while we use github for issues and it might not be an appropriate channel.

```
 * Distribution - (run ```cat /etc/os-release```)

Doesn't matter.

 * Related Application and/or Package Version -  
```
ubuntu-bug

**Issue/Bug Description**

ubuntu-bug doesn't actually submit bugs even though it looks like it will work.  If it is not supported it should be disabled.

**Steps to reproduce (if you know)**

run ubuntu-bug from the command line.

**Expected behavior**

it should submit a bug with logs, or not be available.

**Other Notes**

**Comment by bflanagin** [comment]
Ubuntu-bug is still included in the distribution as of 20.10, on testing the program it does indeed respond with the message:

> The problem cannot be reported:
>
>This does not seem to be an official Pop!_OS package. Please retry after updating the indexes of available packages, if that does not work then remove related third party packages and try again.

**Comment by bflanagin** [comment]
ubuntu-bug is now functioning on 21.10,
 but I believe it is now legacy and not used by Pop!_OS officially.
